### PR TITLE
Mailer 

### PR DESCRIPTION
**Pull request by sameerav**
@sonia-y

### CODE REVIEW

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -11,4 +11,9 @@ def program
 
   def contact
   end
+
+  def send_email
+  	ForteMailer.contact_email(params[:subject], params[:first_name], params[:last_name], params[:email], params[:message]).deliver_now
```
Comment (warrenshen) [inline app/controllers/static_pages_controller.rb]: I would validate these params rather then simply pass them through, but this is fine for now.

File: app/views/forte_mailer/contact_email.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  </head>
+  <body>
+    <p>
+      Hi Forte! You've received the following message from <%= @first_name %> <%= @last_name %>. You can email them back at <%= @email %>.
+    </p>
+    <p>
+      <%= @message %>
+    </p>
+  </body>
+</html>
```
Comment (warrenshen) [inline app/views/forte_mailer/contact_email.html.erb]: New line at EOF!

File: app/views/forte_mailer/contact_email.text.erb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+Hi Forte! You've received the following message from <%= @first_name %> <%= @last_name %>. You can email them back at <%= @email %>.
+
+<%= @message %>
```
Comment (warrenshen) [inline app/views/forte_mailer/contact_email.text.erb]: Same.

**Comment by lilshim** [comment]
CURRENTLY WORKING WOOO

**Pull request merged**